Source/WebCore/css/CSSFontFaceSrcValue.cpp

```diff
@@ -104,6 +104,13 @@ std::unique_ptr<FontLoadRequest> CSSFontFaceSrcResourceValue::fontLoadRequest(Sc
             return nullptr;
     }
 
+    if (!m_technologies.isEmpty()) {
+        for (auto &technology : m_technologies) {
```

Comment:
`auto& technology` I'm surprised the linter doesn't catch this.

(or just `auto`, copying the uint8 is not too costly 😸 )